index.html

```diff
@@ -0,0 +1,227 @@
+          <div class="burger-icon line1"></div>
+          <div class="burger-icon line2"></div>
+          <div class="burger-icon line3"></div>
+        </button>
+        <!--burger menu-->
+        <ul class="burger-menu">
+          <div class="burger-menu-style">
+              <li class="burger-menu-exit-button"><button class="exitButton"></button></li>
+              <li class="burger-menu-homePage-button"><a href="#home-page" class="burger-menu-home-page">Home Page</a></li>
+              <li class="burger-menu-aboutPage-button"><a href="#about-page" class="burger-menu-aboutPage-button">About</a></li>           
+              <li class="burger-menu-aboutPage-other"><a href="" class="burger-menu-aboutPage-other">Join</a></li>
+              <li class="burger-menu-aboutPage-other"><a href="" class="burger-menu-aboutPage-other">Sponsors</a></li>
+              <li class="burger-menu-aboutPage-other"><a href="" class="burger-menu-aboutPage-other">News</a></li> 
+          </div>
+        </ul>
+        <!---->
+        <ul class="desktop-menu">
+          <li><a href="#home-page" class="desktop-menu-home">Home</a></li>
+          <li><a href="#about-page" class="desktop-menu-about">About</a></li>
+          <li><a href="" class="desktop-menu-about">Join</a></li>
+          <li><a href="" class="desktop-menu-about">Sponsors</a></li>
+          <li><a href="" class="desktop-menu-about">News</a></li> 
+        </ul>        
+      </section>
+      <div class="nav-logo-container">
+        <a href="#home-page" class="nav-logo-link">
+          <div class="nav-logo-img"></div>
+            <h1 class="nav-logo">Libra</h1>
+        </a>
+      </div>
+    </div>
+  </header>
+
+  <!--    home page and about me page   -->
+
+  <main>
+
+    <!--    home page    -->
+    
+    <section class="home-page" id="home-page">
+
+      <!--    titles    -->
+    <section class="home-page-intro">
+        <div class="home-page-tittles">
+          <h2 class="home-page-h2">"Security is the future"</h2>
+          <h1 class="home-page-h1">Block-chain first TECH SUMMIT 2023</h1>
+        </div>
+
+        <!--    paragraph    -->
+
+        <div class="home-page-text">
+          <p class="home-page-text-p">Join us into the biggest congrest about block-chain technology
+            and know the new developments and improvements with companies
+            that are already developing applications that will renew the world
+          </p>
+        </div>
+        
+        <!--    where and when descrip    -->
+        
+        <div class="home-page-info">
+          <h2 class="home-page-info-date">2023.10.15 ~ 16</h2>
+          <h3 class="home-page-info-place">see you on: Dubai Exhibition Centre</h3>
+        </div>
+    </section>
+
+      <!--    schedule main program    -->
+
+      <div class="home-page-main-program">        
+        <div class="home-page-main-program-title-content">
+          <h1 class="home-page-main-program-title">Main Program</h1>
```

Comment:
- [ ] Kindly refer to my comments on line `59` on the use of only one `h1` tag for you project 👍